<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1449</width>
    <height>1042</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Teensy TAF 4.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>780</y>
      <width>110</width>
      <height>71</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>780</y>
      <width>110</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="WN_OncheckBox">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>610</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>White Noise ON</string>
    </property>
   </widget>
   <widget class="PlotWidget" name="FFMonitorPlot" native="true">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>40</y>
      <width>361</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="PlotWidget" name="pretemplateView" native="true">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>340</y>
      <width>91</width>
      <height>192</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>4</pointsize>
     </font>
    </property>
   </widget>
   <widget class="PlotWidget" name="DPHistGraph" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>241</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FFT Sampling Rate</string>
    </property>
   </widget>
   <widget class="QComboBox" name="NFFTcomboBox">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="Should ONLY be changed in the Arduino Code. And even then only if you know what you're doing."/>
    </property>
    <item>
     <property name="text">
      <string>256</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1024</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>512</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="SampleRateComboBox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>60</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="Should ONLY be changed in the Arduino Code. And even then only if you know what you're doing."/>
    </property>
    <item>
     <property name="text">
      <string>44100</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>22050</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FFT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uploadtemplateButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>260</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload Template</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>40</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Count</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>180</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FF</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>680</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Recent Match Values</string>
    </property>
   </widget>
   <widget class="QPushButton" name="serialScan">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Scan for Teensy</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Teensy_Com_ComboBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="HitAboveButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>650</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Hit Above</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="HitBelowButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>670</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Hit Below</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FileAndPath_PushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose Save Directory/File</string>
    </property>
   </widget>
   <widget class="QLabel" name="SaveFilePathLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>421</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>./birdname</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="SaveFilePathLabel_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Saving to:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="DirFlagCheckBox">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>580</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Song Flag (e.g. DIR)</string>
    </property>
   </widget>
   <widget class="PlotWidget" name="MagsfromTeensy" native="true">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>340</y>
      <width>111</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="DP_Val">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>530</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Match Value:</string>
    </property>
   </widget>
   <widget class="QLabel" name="SaveFilePathLabel_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Teensy Serial:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RetrieveTemplateButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>290</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Retrieve Template</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="upDateThresholdCheckBox">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>680</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto change Threshold?</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="ThresholdUpdateThresholdspinBox">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>650</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>410</y>
      <width>131</width>
      <height>300</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Teensy Amplitude Minimum</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editMINAMP"/>
       </item>
       <item>
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>Teensy Amplitude Maximum</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editMAXAMP"/>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Match Threshold</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editDP_Thresh"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Minimum FF</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editMINFF"/>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Maximum FF</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editMAXFF"/>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Frequency Threshold</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editFREQ_THRESH"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSpinBox" name="CatchPercentspinBox">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="SaveFilePathLabel_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Teensy Audio::</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Teensy_USB_ComboBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ListeningTextBox">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>730</y>
      <width>250</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="RMSTextBox">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>870</y>
      <width>52</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>870</y>
      <width>52</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>RMS:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="BufferTimeSpinBox">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>890</y>
      <width>190</width>
      <height>22</height>
     </rect>
    </property>
    <property name="suffix">
     <string>     Seconds audio buffering</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="value">
     <number>3</number>
    </property>
   </widget>
   <widget class="PlotWidget" name="FFMonitorPlotTrials" native="true">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>210</y>
      <width>361</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>280</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FF</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>350</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Trials</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="AveragingSpinBox">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>740</y>
      <width>190</width>
      <height>22</height>
     </rect>
    </property>
    <property name="suffix">
     <string> FF estimates to average</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="value">
     <number>3</number>
    </property>
   </widget>
   <widget class="QPushButton" name="TrackFFButton">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>230</y>
      <width>110</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Track Current FF</string>
    </property>
   </widget>
   <widget class="PlotWidget" name="DPvsFFGraph" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>701</y>
      <width>221</width>
      <height>160</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>850</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Recent Match Values</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>760</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>FF</string>
    </property>
   </widget>
   <widget class="QLabel" name="SaveFilePathLabel_6">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>720</y>
      <width>118</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Catch Percentage</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="LineInDoubleSpinBox">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>920</y>
      <width>190</width>
      <height>22</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="prefix">
     <string>Microphone Input Gain (</string>
    </property>
    <property name="suffix">
     <string>dB)</string>
    </property>
    <property name="maximum">
     <double>63.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
    <property name="value">
     <double>55.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="FF_Val">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>540</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FF Value:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editAUDIOAMPTHRESHOLD">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>860</y>
      <width>116</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>840</y>
      <width>116</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>USB Amplitude Threshold</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>740</y>
      <width>51</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>RMS `</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>850</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Recent Match Values</string>
    </property>
   </widget>
   <widget class="PlotWidget" name="RMSvsFFGraph" native="true">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>700</y>
      <width>221</width>
      <height>160</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="AmpCountSpinBox">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>440</y>
      <width>46</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>25</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="DPCountSpinBox">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>520</y>
      <width>46</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>25</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>420</y>
      <width>127</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Between for:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>500</y>
      <width>127</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Above (DP)  for:</string>
    </property>
   </widget>
   <widget class="QLabel" name="RMS_Val">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>550</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>RMS:</string>
    </property>
   </widget>
   <widget class="QLabel" name="TrackedFFLabel">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>230</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tracked FF:</string>
    </property>
   </widget>
   <widget class="QLabel" name="CurrentMeanFFLabel">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>250</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Mean FF:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="isPausedCheckBox">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>780</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause Recording</string>
    </property>
   </widget>
   <widget class="QPushButton" name="templateShiftUpButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>340</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="templateShiftDownButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>370</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>˅</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RetrieveTemplateButton_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>290</y>
      <width>90</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Retrieve Template</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uploadtemplatepreButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>260</y>
      <width>90</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload Template</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>560</y>
      <width>127</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Below (MSE) for:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="DPCountSpinBoxPre">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>580</y>
      <width>46</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>30</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="PreTemplatecheckBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>240</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Pretemplate</string>
    </property>
   </widget>
   <widget class="QLabel" name="DP_ValPre">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>540</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Match Value:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editDP_ThreshPre">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>580</y>
      <width>127</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>570</y>
      <width>127</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Pre Threshold</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editPreDelayMin">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>610</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>600</y>
      <width>127</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Delay Min (ms)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>600</y>
      <width>127</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Delay Max (ms)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editPreDelayMax">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>610</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_28">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>450</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ratio</string>
    </property>
   </widget>
   <widget class="PlotWidget" name="PrePostRatioPlot" native="true">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>380</y>
      <width>361</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>520</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Trials</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1449</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConfig">
    <property name="title">
     <string>Config</string>
    </property>
    <addaction name="actionLoad_Config"/>
    <addaction name="actionSave_Config"/>
   </widget>
   <addaction name="menuConfig"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_Config">
   <property name="text">
    <string>Load Config</string>
   </property>
  </action>
  <action name="actionSave_Config">
   <property name="text">
    <string>Save Config</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header location="global">pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
